tests/decap/test_decap.py

```diff
@@ -33,6 +33,9 @@ def ttl_dscp_params(duthost, supported_ttl_dscp_params):
     if "uniform" in supported_ttl_dscp_params.values() and ("201811" in duthost.os_version or "201911" in duthost.os_version):
         pytest.skip('uniform ttl/dscp mode is available from 202012. Current version is %s' % duthost.os_version)
 
+    if supported_ttl_dscp_params['dscp'] == 'pipe' and duthost.facts['asic_type'] in ['cisco-8000']:
```

Comment:
is this skipping only dscp - pipe or ttl as well?

Reply:
@prsunny : In test_decap.py script, the current tested combinations are:

supported_ttl_dscp_params : DSCP: uniform, TTL:pipe -- we support this case
supported_ttl_dscp_params : DSCP: pipe, TTL:pipe -- we skip this case

To answer your question, we only skip for dscp - pipe.